tests/test_plotting.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `seamm_util` package, plotting module."""
+
+from seamm_util import Figure
+import jinja2
+import os
+# import pytest
+
+# Create the Jinja environment with the path we want
+results_path = os.path.join(os.path.dirname(__file__), 'results')
+template_path = os.path.join(os.path.dirname(__file__), 'templates')
+env = jinja2.Environment(loader=jinja2.FileSystemLoader(template_path))
+
+
+def test_simple_plot():
+    """Testing that we can create a simple plot."""
+    fig = Figure(
+        jinja_env=env, template='line.html_template', title='Simple plot'
+    )
+
+    plot = fig.add_plot('Energies')
+
+    x_axis = plot.add_axis('x', label='Time (ps)')
```

Comment:
Is there any way we can make it so that the people creating the graph do not have to explicitly create the axes themselves? It could be an option to create your own, but having a default might be nice.

Reply:
It would be straightforward; however, the axis would be unlabeled. It is a compromise between generality and ease of use -- the basic graph is a bit more complicated, but it is straightforward to have 3, 4, ... n axes on a single plot. 3 or 4 are fairly common with two y-axes, one on left and one on right. And sometimes two x-axes, top and bottom. Occasionally you see graphs with 3 or 4 y-axes. 

Those are easy to set up at the moment, but would be complex with a more specialized solution that made the base case easy.

I suggest that this should be revisited when we have more experience. Certainly the current approach should be supported forever, but maybe there could be a SimplePlot or similar derived from Plot....?